Makefile

```diff
@@ -14,7 +14,7 @@ lint:
 	flake8 . # && isort --check-only --recursive .
 	gulp lint --production
 	!(find . -wholename '**migrations**/*.py' -print0 | xargs -0 grep 'RemoveField\|DeleteModel\|AlterModelTable\|AlterUniqueTogether\|RunSQL\|RunPython\|SeparateDatabaseAndState' | grep -v '# manually-reviewed') || { echo "Migrations need to be manually reviewed!"; exit 1; }
-	!(cd example_project && python3 manage.py makemigrations --dry-run --exit) || { printf "Migrations need to be created. Try: \n\t python3 manage.py makemigrations\n"; exit 1; }
+	cd example_project && python3 manage.py makemigrations --check || { printf "Migrations need to be created. Try: \n\t python3 manage.py makemigrations\n"; exit 1; }
```

Comment:
Have you tested that this still fails/stops the build as desired?

Reply:
I've tested this on my machine in the following way:
- ran `python3 manage.py makemigrations --dry-run --check || { printf "Migrations need to be created. Try: \n\t python3 manage.py makemigrations\n"; }`. The output was `No changes detected`.
- updated the model field 
- ran `python3 manage.py makemigrations --dry-run --check || { printf "Migrations need to be created. Try: \n\t python3 manage.py makemigrations\n"; }` again. The output was `Migrations need to be created.`

Added `--dry-run` cause it was creating migration files.